Network/Converter/NetworkObjectState.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Network.Converter
```

Comment:
Rename back to ObjectState. This change is too critical due to the class scope. (public)

Reply:
K